README.md

```diff
@@ -410,37 +316,26 @@ enum Mixed {
 export enum Zoo { lion, tiger, bear };
 export default enum { up, down, left, right };
 
-// All enum declaration constructors inherit from a built in %Enum% object with common 
-// static methods:
-
-// You can test for membership using `%Enum%.has()`:
-Numbers.has(0) === true; // for underlying values
-Numbers.has("one") === true; // for enum member names
-Numbers.has(Numbers.two) === true; // for enum values
-Numbers.has(Numbers(9)) === false; // even though coerced
-Numbers.has(9) === false;
-Numbers.has(Color.blue) == false; // even though the underlying values are the same
-
-// You can test for enum branding using `%Enum%.is()`:
-Numbers.is(0) === false;
-Numbers.is("one") === false;
-Numbers.is(Numbers.two) === true;
-Numbers.is(Numbers(9)) === true; // even though not defined
-Numbers.is(Color.blue) === false;
-
-// You can lookup an enum value by key or value using `%Enum%.forKey()` and 
-// `%Enum%.forValue()`, respectively:
-enum AlphaBeta {
+// You can test for name membership using `Enum.has()`
+Enum.has(Numbers, "one") === true
+Enum.has(Numbers, "five") === false
+
+// You can test for value membership using `Enum.hasValue()`:
+Enum.hasValue(Numbers, 0) === true
+Enum.hasValue(Numbers, 9) === false
+
+// You can convert enums between names and values using 
+// `Enum.parse` and `Enum.format`, respectively.
+enum AToB {
     a = "b",
     b = "a",
 }
 
-AlphaBeta.forKey("a") === AlphaBeta.a;
-AlphaBeta.forKey("b") === AlphaBeta.b;
-AlphaBeta.forValue("a") === AlphaBeta.b;
-AlphaBeta.forValue("b") === AlphaBeta.a;
+Enum.parse(AToB, "a") === AToB.a // "b"
+Enum.parse(AToB, "b") === AToB.b // "a"
 
-// There is a global `Enum` object with additional capabilities:
+Enum.format(AToB, AToB.a) === "b"
```

Comment:
parse and format seem like strange names for functions that are essentially looking up an enum name by value, and getting the valueOf an enum name.

Reply:
Their utility is expanded when you consider the bit flags case of `Enum.parse(Flags, "member1, member2")`

Reply:
How would one do bit flags with this proposal?

Reply:
The primary mechanism I considered would be through an `Enum.flags` decorator, which is mentioned in the proposal. 

Another way would be through the `extends` clause and some other magic:

```js
function BitFlags(value) { return value; }
Object.setPrototypeOf(BitFlags, Number);
Object.setPrototypeOf(BitFlags.prototype, Number.prototype);
BitFlags[Symbol.toEnum] = function (key, autoValue) {
  const value = autoValue.value;
  autoValue.value <<= 1;
  return value;
}
BitFlags.makeFlags = function(enumObj) {
  // override @@formatEnum and @@parseEnum
  Object.defineProperty(enumObj, Symbol.formatEnum, { value: function (value) { ... } });
  Object.defineProperty(enumObj, Symbol.parseEnum, { value: function (value) { ... } });
}

enum MyFlagsEnum extends BitFlags {
  Zero,
  One,
  Two,
  Four
}
BitFlags.makeFlags(MyFlagsEnum);
```

The decorator would just do that work for you.

Reply:
The `BitFlags` version above is probably not something I'd want to add to the spec, and using it is a bit cumbersome. The `@Enum.flags` decorator approach is much cleaner, and is modeled somewhat after [.NET's FlagsAttribute](https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=netframework-4.7.1#examples). 

Since this version of `enum` doesn't use its own custom primitive type, we can't override the `toString()` behavior as the .NET version does. And since we generally don't want to clutter an _enum object_ with declarations that can collide with member names they might choose, we have to put this somewhere else, like on `Enum`.

For `Enum.format`, I originally called it `Enum.getName`, but switched to `Enum.format` since it could do more than just get the name. I might end up with both, since the .NET `Enum` type supplies both as static methods of the `Enum` type (the .NET version of `Enum` is a strong inspiration for some of the design in this proposal).